docs/commands/lint.md

```diff
@@ -2,239 +2,378 @@
+:::warning Important
+Unlike the `bundle` command, `lint` doesn't execute decorators. To learn more about preprocessors and rules, see the [custom rules](../resources/custom-rules.md) page.
 :::
 
-## Usage
+## Options
 
-```bash
-openapi lint
-openapi lint <entrypoints>...
-openapi lint [--max-problems=<n>] [--config=<path>] [--format=<value>]
-openapi lint [--generate-ignore-file] [--help]
-openapi lint --version
+Always the over-achiever, `lint` has many options which you can execute alone, or combine with others in a single command:
+
+```bash Standalone
+redocly lint
+redocly lint <entrypoints>...
+redocly lint --config=<path>
+redocly lint --extends=<value>
+redocly lint --format=<value>
+redocly lint --generate-ignore-file
+redocly lint --help
+redocly lint --max-problems=<n>
+redocly lint --skip-preprocessor=<preprocessor>
+redocly lint --skip-rule=<rule>,<rule>
+redocly lint --version
 ```
 
-## Options
+```bash Combined
+redocly lint [--max-problems=<n>] [--config=<path>] [--format=<value>]
+```
 
-Option                   | Type      | Required    | Default     | Description
--------------------------|:---------:|:------------:|:-----------:|------------
-`entrypoints`            | `array`   | no           | -           | Array of API definition filenames that need to be linted. See [the Entrypoints section](#entrypoints) for more options
-`--config`               | `string`  | no           | -           | Specify path to the [configuration file](#custom-configuration-file)
-`--extends`              | `array`   | no           | -           | [Extend a specific configuration](#extend-configuration) (defaults or config file settings)
-`--format`               | `string`  | no           | `codeframe` | Format for the output.<br />**Possible values:** `codeframe`, `stylish`, `json`, `checkstyle`
-`--generate-ignore-file` | `boolean` | no           | -           | [Generate ignore file](#generate-ignore-file)
-`--help`                 | `boolean` | no           | -           | Show help
-`--max-problems`         | `number`  | no           | 100         | Truncate output to display the specified [maximum number of problems](#max-problems)
-`--skip-preprocessor`    | `array`   | no           | -           | Ignore certain preprocessors. See the [Skip preprocessor or rule section](#skip-preprocessor-or-rule) below
-`--skip-rule`            | `array`   | no           | -           | Ignore certain rules. See the [Skip preprocessor or rule section](#skip-preprocessor-or-rule) below
-`--version`              | `boolean` | no           | -           | Show version number
+| Option | Type | Description |
+|---|---|---|
+| entrypoints | array | Lets you [pass the entrypoints](#how-to-use-entrypoints-to-check-all-or-selected-definitions) (locations) of specific definition files in the command so `lint` only checks those files. |
+| --config | string | Lets you [specify a path to a custom config file](#how-to-reference-a-custom-config-file). |
```

Comment:
I'm not sure "Lets you" makes sense in the context of CLI options.

I'm used to a more declarative style:

https://awscli.amazonaws.com/v2/documentation/api/latest/reference/index.html
https://cli.github.com/manual/gh_auth_login
https://curl.se/docs/manpage.html
https://git-scm.com/docs/git-switch